deploy/examples/create-external-cluster-resources.py

```diff
@@ -213,7 +216,7 @@ def gen_arg_parser(cls, args_to_parse=None):
 
         upgrade_group = argP.add_argument_group('upgrade')
         upgrade_group.add_argument("--upgrade", action='store_true', default=False,
-                                   help="Upgrades the 'csi-user'(For example: client.csi-cephfs-provisioner) with new permissions needed for the new cluster version and older permission will still be applied." +
+                                   help="Upgrades the cephCSIKeyrings(For example: client.csi-cephfs-provisioner) with new permissions needed for the new cluster version and older permission will still be applied." +
```

Comment:
```suggestion
                                   help="Upgrades the 'csi-user'(For example: client.csi-cephfs-provisioner) with new permissions needed for the new cluster version and older permission will still be applied." +
```
seems good to me as it csi ceph user

Reply:
It's actually the keyring only, the user is created below with the help of this keyring.